maven-model/src/main/mdo/maven.mdo

```diff
@@ -1087,8 +1087,10 @@
           <version>3.0.0+</version>
           <description>
             <![CDATA[
-            The version of the dependency, e.g. <code>3.2.1</code>. In Maven 2, this can also be
-            specified as a range of versions.
+            The version requirement of the dependency, e.g. <code>3.2.1</code> for soft version requirement: the effective version will be resolved based on the usage context.
```

Comment:
I can follow this, but you really need to be pretty deep in Maven to grok terms like "soft version" and "effective version". Maybe something like 

The version requirement of the dependency, e.g. <code>3.2.1</code>. The actual version added to the classpath at runtime and compile time may change depending on other dependencies in the tree.

Reply:
ok, fair
I'll rework it: feedback appreciated